src/subscribers/websocket.c

```diff
@@ -1091,6 +1091,8 @@ static ngx_buf_t *websocket_inflate_message(full_subscriber_t *fsub, ngx_buf_t *
   z_stream      *strm;
   int            rc;
   ngx_buf_t     *outbuf;
+	
+  *result = 0;
```

Comment:
It seems `*result` is still uninitialized if `NGX_ZLIB` is not defined (perhaps that does not happen, but it still seems wrong to write code that will not work).

Reply:
oops... fixed